.github/workflows/build_and_test.yml

```diff
@@ -31,3 +32,91 @@ jobs:
           file: ./*/target/site/jacoco/jacoco.xml
           flags: unittests
           fail_ci_if_error: true # optional (default = false)
+      - name: Upload diktat jar
+        uses: actions/upload-artifact@v2
+        with:
+          name: diktat-ruleset
+          path: diktat-ruleset/target/diktat-*.jar
+          # no need to store artifact longer, it is used only by dependant jobs
+          retention-days: 1
+
+  run_diktat_from_CLI:
+    name: Run diktat via CLI
+    runs-on: ${{ matrix.os }}
+    needs: build_and_test_with_code_coverage
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+
+    steps:
+      - uses: actions/checkout@v2.3.3
+
+      - name: Setup environment variables
+        run: |
+          cat pom.xml | grep "<version>.*</version>" | head -1 | awk -F'[><]' '{ printf "DIKTAT_VERSION=%s\n",$3 }' >> $GITHUB_ENV
+          cat pom.xml | grep "<ktlint.version>.*</ktlint.version>" | head -1 | awk -F'[><]' '{ printf "KTLINT_VERSION=%s\n",$3 }' >> $GITHUB_ENV
+        shell: bash
+
+      - name: Setup environment
+        run: |
+          curl -sSLO https://github.com/pinterest/ktlint/releases/download/${{ env.KTLINT_VERSION }}/ktlint && chmod a+x ktlint
+        shell: bash
+
+      - name: Download diktat jar
+        uses: actions/download-artifact@v2
+        with:
+          name: diktat-ruleset
+
+      - name: Run diKTat from cli
```

Comment:
It's displayed in check name, like `Build and test / Run diktat via CLI (ubuntu-latest)`. For windows it's explicitly stated in yml, because there are steps that are only executed on windows (to use different shells).